static/src/javascripts/projects/common/views/svgs.js

```diff
@@ -5,6 +5,7 @@ define([
     'inlineSvg!svgs/comment-16!icon',
     'inlineSvg!svgs/marque-36!icon',
     'inlineSvg!svgs/marque-54!icon',
+    'inlineSvg!svgs/arrow!icon',
```

Comment:
could we chat about naming conventions when you have a sec? not a blocker on this though